docs/guides/how-to-add-contest-table-provider.md

```
@@ -268,423 +144,271 @@ class EDPCProvider extends ContestTableProviderBase {
+    const provider = new MyNewProvider(ContestType.MY_NEW);
     const config = provider.getDisplayConfig();
     expect(config.isShownHeader).toBe(true);
   });
 });
 ```
 
----
+### モックデータ準備
 
-## 教訓: よくあるミス
+**ステップ1: データソース確認**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)